### ISSUE TITLE
please remove pyflakes and rope from the .zip files

### ISSUE THREAD

**Issue by spyder-issues-migrator** [spyder-ide/spyder#1128]
_From [pi...@synchrotron-soleil.fr](https://code.google.com/u/111976521841338120241/) on 2012-07-31T15:07:39Z_

Hello

since the 2.1.11 version it seems that pyflakes and rope are bundeled with the spyder sources.

From the point of view of the packager this is a problem.
I need to check for the licences of all the relevant informations into the Debian copyright file.

is it possible to remove theses libraries from the .zip file.

thanks,

your Debian Maintainer

_Original issue: http://code.google.com/p/spyderlib/issues/detail?id=1128_

**Comment by spyder-issues-migrator** [comment]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2013-01-24T11:58:27Z_

Sorry, Frédéric, I completely forgot about this request.
I'll handle it as soon as I can.

**Status:** Accepted  
  **Owner:** pierre.raybaut  
  **Labels:** Cat-Miscelleneous

**Comment by spyder-issues-migrator** [comment]
_From [jed.lud...@gmail.com](https://code.google.com/u/114333776555678317161/) on 2013-01-24T14:38:45Z_

Frédéric, as a general question first, from the application's point of view, it is sometimes convenient to bundle a specific version of a particular package that shares a common license style so that the end users don't get bitten by incompatible versions and relax inter-package dependency. What is the preferred method to handle this from a Debian perspective? 

I guess this question would apply to issue #1230 as well.

**Comment by spyder-issues-migrator** [comment]
_From [ccordoba12](https://code.google.com/u/ccordoba12/) on 2013-01-24T17:35:29Z_

**Labels:** MS-v2.2

**Comment by spyder-issues-migrator** [comment]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2013-01-26T04:12:42Z_

Frédéric is right: the Debian ecosystem can provide (transparently for the end user) the right versions of the libraries as specified in README.

The reason why rope and pyflakes were accidently included in the source distribution (.zip package) is that I chose to include them in the Windows distutils installers, as I did before with Python(x,y) plugin. But, my test condition to bundle rope/pyflakes or not in the package was wrong (I only tested if I was on Windows or not... and since I'm building the source package from Windows as well...).
